### PR TITLE
i18n(fr): update `getting-started.mdx`

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

This PR updates the french version of the `getting-started.mdx` page with the changes of #1026.

### CODE REVIEW

File: docs/src/content/docs/fr/getting-started.mdx [hunk 1]
```diff
@@ -45,26 +46,71 @@ Essayez Starlight dans votre navigateur :
...
+
+Starlight prend en charge la création de contenu en Markdown et MDX sans aucune configuration requise.
+Vous pouvez ajouter la prise en charge de Markdoc en installant [l'intégration Astro Markdoc](https://docs.astro.build/fr/guides/integrations-guide/markdoc/) expérimentale.
+
+#### Ajouter des pages
+
+Ajoutez de nouvelles pages à votre site en créant des fichiers `.md` ou `.mdx` dans `src/content/docs/`.
+Utilisez des sous-dossiers pour organiser vos fichiers et créer plusieurs segments de chemin.
+
+Par exemple, la structure de fichiers suivante génére des pages à `example.com/hello-world` et `example.com/guides/faq` :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:96]: Decided to not translate "hello world" based on the content of [this page](https://fr.wikipedia.org/wiki/Hello_world) which indicates that we can translate it literally but it's not really used or common.

File: docs/src/content/docs/fr/getting-started.mdx [hunk 2]
```diff
@@ -45,26 +46,71 @@ Essayez Starlight dans votre navigateur :
...
+- src/
+  - content/
+    - docs/
+      - guides/
+        - faq.md
+      - hello-world.md
+
+</FileTree>
+
+#### Frontmatter avec sûreté du typage
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:111]: [Reference](https://fr.wikipedia.org/wiki/S%C3%BBret%C3%A9_du_typage)

File: docs/src/content/docs/fr/getting-started.mdx [hunk 3]
```diff
@@ -93,37 +141,34 @@ Starlight est une intégration Astro, et est mis à jour comme toute intégratio
...
 ```sh
-# mettre à jour Starlight avec yarn
 yarn upgrade @astrojs/starlight --latest
 ```
 
 </TabItem>
 </Tabs>
 
-Vous pouvez voir une liste complète des modifications apportées à chaque version dans [le journal des modifications de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md).
+Consultez le [journal des modifications de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) pour une liste complète des modifications apportées à chaque version.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:164]: Reference for the "changelog" translation: https://fr.wikipedia.org/wiki/Changelog

**Review by delucis — APPROVED** [review]
Looks great! Thanks for the careful translation @HiDeoo 💜

**Pull request merged**